src/api_server/swagger/firecracker.yaml

```diff
@@ -960,6 +973,13 @@ definitions:
         format: "169.254.([1-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-4]).([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])"
         default: "169.254.169.254"
         description: A valid IPv4 link-local address.
+      version:
+        description: Enumeration indicating the MMDS version configured.
+        type: string
+        enum:
+          - V1
+          - V2
+        default: V1
```

Comment:
shouldn't V2 be default?

Reply:
The token generation logic is not integrated yet, I will set default to V2 as part of the integration PR, on top of this one.